token/js/src/extensions/extensionType.ts

```diff
@@ -195,11 +216,15 @@ function getLen(extensionTypes: ExtensionType[], baseSize: number): number {
     }
 }
 
-export function getMintLen(extensionTypes: ExtensionType[]): number {
-    return getLen(extensionTypes, MINT_SIZE);
+export function getMintLen(
+    extensionTypes: ExtensionType[],
+    variableLengthExtensions?: { [E in ExtensionType]?: number }
```

Comment:
nit: can this also take a default?
```suggestion
    variableLengthExtensions: { [E in ExtensionType]?: number } = {}
```

Reply:
Force-pushed this in 👊🏼